src/redux/actions/samples/createSampleFile.js

```diff
@@ -41,20 +55,9 @@ const createSampleFile = (
       },
     );
 
-    dispatch({
-      type: SAMPLES_FILE_UPDATE,
-      payload: {
-        sampleUuid: sampleId,
-        lastModified: updatedAt,
-        fileName: fileNameForApiV1[type],
-        fileDiff: fileForApiV1,
-      },
-    });
-
     return signedUrl;
   } catch (e) {
-    // Can't update the upload status because we didn't even get to create the sample file
-    handleError(e, endUserMessages.ERROR_BEGIN_SAMPLE_FILE_UPLOAD);
+    dispatch(updateSampleFileUpload(experimentId, sampleId, type, UploadStatus.UPLOAD_ERROR));
```

Comment:
So if it fails you try to upload it again and if that fails again then the error message is outputted to the user? 
If thats the case why dont you have ` dispatch(updateSampleFileUpload(experimentId, sampleId, type, UploadStatus.UPLOAD_ERROR));` in this try too?

Reply:
That's the change, I am doing the dispatch upload error

Reply:
I don't understand exactly the suggestions here

Reply:
Oh I thought the two actions createsamplefile and updatesamplefile could be one, but I saw that there is a difference